Resources/Prototypes/Body/Organs/Animal/animal.yml

```diff
@@ -113,6 +122,9 @@
     groups:
     - id: Alcohol
       rateModifier: 0.1
+  - type: Item
+    size: Normal
```

Comment:
i think would be better to leave normal for all organs

Reply:
makes sense

Reply:
This does beg the question though, can you fit a spessman in a duffelbag?